src/components/navbar.svelte

```diff
@@ -1,26 +1,23 @@
 <script>
-    import { players, count } from './stores.js';
     import Player from './player.svelte'
     import Paper from '@smui/paper';
     import { goto } from '@sveltech/routify';
     import Menu from '@smui/menu';
+    import Select, {Option} from '@smui/select';
     import List, {Item, Separator, Text, PrimaryText, SecondaryText, Graphic} from '@smui/list';
-    
+    import TopAppBar, {Row, Section, Title} from '@smui/top-app-bar';
+    import IconButton from '@smui/icon-button';
+    import { currPlayer, players, setCurrentPlayer } from '../playerService.js';
+
     let menu;
     let options;
-    let playerProfile = false;
+    let showProfile = false;
+    let playerChoice = '';
+    $: playerChoice && setCurrentPlayer(playerChoice);
 </script>
 
 <style>
-    a {
```

Comment:
it's great when you can remove code, nice cleanup!

Reply:
Hack it away I say!